### PR TITLE
Add test requirements to CONTRIBUTING.md

### LINKED ISSUE #21: need to add tests requirements to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mihaitusa**
Fixes #21 
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173 Please review. Thanks!

**Comment by meshipeer** [comment]
fixes #21

**Comment by SamMarkowitz** [comment]
change "from a plugin to another" to "from one plugin to another"

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,6 +75,38 @@ To make a pull request you will need a GitHub account. For help, see GitHub's do
 
 Normally, all pull requests must include tests that test your change. Occasionally, a change will be very difficult to test. In those cases, please include a note in your commit message explaining why tests are not included.
 
+## Unit tests for score-actions
+
+JUnit is a program that can be used to perform unit testing of software by writing test cases in Java.
```
Comment (gituser173) [inline CONTRIBUTING.md]: is a unit testing framework..

**Comment by gituser173** [comment]
Looks :+1:

**Pull request merged**